src/client/rpc.rs

```diff
@@ -61,8 +62,8 @@ impl<T, E: Code> Code for Result<T, E> {
 impl Code for RpcClientError {
     fn code(&self) -> i32 {
         match self {
+            RpcClientError::InvalidPayload(_) => INVALID_PAYLOAD,
```

Comment:
are there other rpc error types? also I wonder if the code is already in error for `RpcClientError::InvalidPayload(_)` instead of having to manually specify the code as a constant

Reply:
The code will default to `INTERNAL_ERROR` I believe. I can add the other error types if you like (Io, jwt)

Reply:
I think the code span will be `13` currently for any of the spans with internal server error.

Reply:
I meant the error inside `RpcClientError::InvalidPayload(e)` - I thought there were default JSON RPC error codes encoded

Reply:
otherwise is fine to me, can be added as a follow up